pages/file-feed.tsx

```diff
@@ -75,18 +75,29 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     };
   }
 
-  const actions = await getActions(
+  // console.log("space", space);
+
+  const eventActions = await getActions(
     token.organizationId,
     ActionType.FileFeedEvent
   );
 
+  const syncActions = await getActions(
+    token.organizationId,
+    ActionType.SyncFilefeedRecords
+  );
+
+  const actions = [...eventActions, ...syncActions].sort((a, b) => {
+    return a.createdAt > b.createdAt ? -1 : 1;
+  });
+
   const events = actions.map((a) => {
     const metadata = a.metadata as {
       topic: string;
     };
 
     return {
-      topic: metadata.topic,
+      topic: metadata.topic || null,
```

Comment:
should always have a topic here I think?

Reply:
removed